### PR TITLE
Ignore js style linting in typescript examples

### PR DESCRIPTION
**Pull request by samccone**

### CODE REVIEW

File: .jscsrc [hunk 1]
```diff
@@ -34,6 +34,8 @@
 		"examples/polymer/elements/elements.build.js",
 		"examples/js_of_ocaml/js/*.js",
 		"examples/typescript-react/js/*.js",
+		"examples/typescript-angular/js/*.js",
+		"examples/typescript-backbone/js/*.js",
```
Comment (arthurvr) [inline .jscsrc]: Wouldn't `"examples/typescript-*/js/*.js",` work?

Comment (samccone) [inline .jscsrc]: I think it might

**Comment by samccone** [comment]
good call @arthurvr 

screen shot 2015-10-19 at 12 50 35 pm

**Comment by passy** [comment]
:+1:

**Pull request merged**